includes/background-jobs/class-sensei-scheduler-action-scheduler.php

```diff
@@ -59,16 +57,21 @@ public function schedule_job( Sensei_Background_Job_Interface $job, $time = null
 	public function run( Sensei_Background_Job_Interface $job, $completion_callback = null ) {
 		$this->current_job = $job;
 
-		$this->schedule_job( $job );
+		// Schedule action 60 seconds in the future as fallback.
+		$this->schedule_job( $job, time() + 60 );
 
 		$job->run();
 
-		if ( $job->is_complete() ) {
-			$this->cancel_scheduled_job( $job );
+		// Clean scheduled jobs (remove fallback).
+		$this->cancel_scheduled_job( $job );
 
+		if ( $job->is_complete() ) {
 			if ( is_callable( $completion_callback ) ) {
 				call_user_func( $completion_callback );
 			}
+		} else {
+			// Re-schedule immediately.
+			$this->schedule_job( $job, time() );
```

Comment:
```suggestion
			$this->schedule_job( $job );
```

Same behavior, but just a suggestion to use the conditional from the `schedule_job` that uses `time()` as default.

Reply:
I wanted to make it explicit here. But if you think it is more reasonable to use the default argument I am ok with that. (There is already a comment making it explicit so.. 👍 )